### ISSUE TITLE
Letter 't' Still Doesn't Look Right

### ISSUE THREAD

**Issue by matsliapich** [tokotype/PlusJakartaSans#11]
The fixed version made it better, but I believe that the ascender of letter 't' is still not high enough.

I provided Product Sans, Manrope, Circular std., and fixed version of +Jakarta Sans for reference:
![20201218_121710](https://user-images.githubusercontent.com/75990540/102644473-e159ec80-412e-11eb-992c-3aab92b0b5a3.png)

**Comment by matsliapich** [comment]
Suggested change in red:
![20201218_122431](https://user-images.githubusercontent.com/75990540/102644658-2bdb6900-412f-11eb-9761-63580b75e0c1.png)
Before:
![20201218_122701](https://user-images.githubusercontent.com/75990540/102644688-3ac21b80-412f-11eb-954f-3d7d86a5505f.png)
After:
![20201218_122650](https://user-images.githubusercontent.com/75990540/102644702-4281c000-412f-11eb-8b4b-7d8edcfd86d4.png)

**Comment by somegoodpixels** [comment]
This is a matter of preference, not a bug. There are plenty of well-crafted fonts that have unconventional spacing or character designs. If the designer sees these changes as an improvement, then great! If not... fork the repo and build your own variation.

**Comment by matsliapich** [comment]
I totally agree with you! It was meant as a suggestion and I'm fine with whichever way the development chooses to go:)

**Comment by matsliapich** [comment]
Having a lower ascender might improve the reading "flow" of the font, and therefore improve it's readability.

On the other hand, higher ascender is more aesthetically pleasing (in my opinion).

The other option would be to find a middle ground so it's readable and aesthetically pleasing.

By the way, thank you for the font! It's definitely my go-to from now on.
